tests/charms/mongodb_test_charm/metadata.yaml

```diff
@@ -17,9 +17,22 @@ series:
   - noble
 
 storage:
-  mongodb:
+  archive:
     type: filesystem
-    location: /var/snap/charmed-mongodb/common
+    location: /var/snap/charmed-mongodb/common/archive
+    description: storage for local backup files.
+  data:
+    type: filesystem
+    location: /var/snap/charmed-mongodb/common/var/lib/mongodb
+    description: storage for MongoDB data.
+  logs:
+    type: filesystem
+    location: /var/snap/charmed-mongodb/common/var/log/mongodb
+    description: storage for logfiles.
+  temp:
+    type: filesystem
+    location: /var/snap/charmed-mongodb/common/temp
```

Comment:
This should be `/tmp`, but i'm not entirely sure if the snap can access `/tmp` of the host or not

Reply:
It's given a /tmp on a custom path: `/tmp/snap-private-tmp/snap.charmed-mongodb/tmp/`

This is where the socket files exist for example.
For the snap user, this is seen as `/tmp`:
```
$ sudo du -h
4.0K	./snap-private-tmp/snap.charmed-mongodb/tmp
8.0K	./snap-private-tmp/snap.charmed-mongodb

Reply:
Oh maybe because the dir already exists and is created by the snap immediately :thinking: